src/app/code/Komoju/Payments/Controller/HostedPage/Redirect.php

```diff
@@ -165,7 +168,8 @@ private function createReturnUrl($orderId)
         $secretKey = $this->config->getSecretKey();
         $endpoint = 'komoju/hostedpage/postsessionredirect?order_id=' . $orderId;
         $hmac = hash_hmac('sha256', $endpoint, $secretKey);
-        $returnUrl = $this->_url->getUrl($endpoint .= '&hmac='.$hmac);
+        $returnUrl = $this->_url->getUrl($endpoint .= '&hmac_magento='.$hmac);
+        $returnUrl = $this->_url->getUrl($endpoint);
```

Comment:
Why are we setting `$returnUrl` twice here? Is that a typo, or is it some PHP magic ✨ I'm not aware of?

Reply:
Ah! It's.. typo magic 😨 Thanks for pointing it out!